data/repos.yml

```diff
@@ -847,6 +1067,12 @@
   team: "#tech-interaction-and-personalisation"
   production_hosted_on: aws
 
+# Propose deleting - it's a fork far behind the original.
+# I don't see it referenced anywhere (unlike the CKAN forks).
+# And it does appear (based on citations used in [this PR](https://github.com/alphagov/govuk-puppet/pull/9200))
+# that the original is the thing being used now, not the fork.
+- repo_name: logstasher
```

Comment:
Hmmm I may defer to @rtrinque again here - can you confirm we don't appear to be using this?